### DISCUSSION TITLE
wallet creation

### DISCUSSION THREAD

**Discussion by heczlu** [qlcchain/go-qlc#1398] [Ideas]
with increasing cases from people with people "wallet problem" which is ending up with lost private key I would like to suggest a step in wallet creation in which after you get generated 24 word passphrase in step after you would need to write like 5 words from that passphrase in correct possition to be able to create wallet.

**Comment by nikes90** [comment]
I agree on this!
A kind of confirmation asking user ID he has copied n provide info they - he needs to use the words in next step for 5 words matching!
